config/mime-multipart.go

```diff
@@ -0,0 +1,160 @@
+package config
+
+import (
+	"io"
+	"io/ioutil"
+	"mime"
+	"mime/multipart"
+	"net/mail"
+	"os"
+	"strings"
+
+	"github.com/coreos/coreos-cloudinit/pkg"
```

Comment:
Same questions as above I guess. Any objection to placing this code in the initialize package then?